cli/tests/pcluster/config/test_validators.py

```diff
@@ -2018,3 +2019,39 @@ def test_fsx_ignored_parameters_validator(mocker, section_dict, expected_error):
         assert_that(errors[0]).matches(expected_error)
     else:
         assert_that(errors).is_empty()
+
+
+@pytest.mark.parametrize(
+    "section_dict, expected_error",
+    [
+        ({"volume_type": "standard", "volume_size": 15}, None),
+        ({"volume_type": "standard", "volume_size": 0}, "The volume size of standard must be at least 1 GiB"),
+        ({"volume_type": "standard", "volume_size": 1025}, "The volume size of standard must be at most 1 TiB"),
+        ({"volume_type": "io1", "volume_size": 15}, None),
+        ({"volume_type": "io1", "volume_size": 3}, "The volume size of io1 must be at least 4 GiB"),
+        ({"volume_type": "io1", "volume_size": 16385}, "The volume size of io1 must be at most 16 TiB"),
+        # TODO Uncomment these lines agter making io2 supported
+        # ({"volume_type": "io2", "volume_size": 15}, None),
+        # ({"volume_type": "io2", "volume_size": 3}, "The volume size of io2 must be at least 4 GiB"),
+        # ({"volume_type": "io2", "volume_size": 16385}, "The volume size of io2 must be at most 16 TiB"),
+        ({"volume_type": "gp2", "volume_size": 15}, None),
+        ({"volume_type": "gp2", "volume_size": 0}, "The volume size of gp2 must be at least 1 GiB"),
+        ({"volume_type": "gp2", "volume_size": 16385}, "The volume size of gp2 must be at most 16 TiB"),
+        ({"volume_type": "st1", "volume_size": 500}, None),
+        ({"volume_type": "st1", "volume_size": 20}, "The volume size of st1 must be at least 500 GiB"),
+        ({"volume_type": "st1", "volume_size": 16385}, "The volume size of st1 must be at most 16 TiB"),
+        ({"volume_type": "sc1", "volume_size": 500}, None),
+        ({"volume_type": "sc1", "volume_size": 20}, "The volume size of sc1 must be at least 500 GiB"),
+        ({"volume_type": "sc1", "volume_size": 16385}, "The volume size of sc1 must be at most 16 TiB"),
+    ],
+)
+def test_ebs_volume_type_size_validator(mocker, section_dict, caplog, expected_error):
+    config_parser_dict = {"cluster default": {"ebs_settings": "default"}, "ebs default": section_dict}
+    utils.assert_param_validator(mocker, config_parser_dict, expected_error)
+
+
+def test_volume_type_size_match():
```

Comment:
is there a better name for this function? maybe something that expresses more clearly the intention of this test?

also, can you add a single-sentence docstring to the function explaining why it exists?

Reply:
Yes! What about change the function name to "ebs_allowed_values_all_have_volume_size_bounds"?

Reply:
perfect!